README.md

```diff
@@ -0,0 +1,193 @@
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#Live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Metrics-Webapp] <a name="about-project"></a>
+
+**Mtrics-Webapp** Is a web application that shows information about countries all around the world.
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+     <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/Javascript">Javascript</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[The homepage allows users to access real-time countries data for various countries.]**
+- **[The detail page provides insights on the countries data for the selected country.]**
+- **[A filtering feature enables users to search for specific country.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Live demo <a name="Live-demo"></a>
+
+
+[Live link](frolicking-brigadeiros-17734d.netlify.app)
```

Comment:
- [ ] I would like to advise you to please double check your `live` link before asking for the review, I tried to see your live project, and your provided link is correct but because you forgot to add the `https://` before the `link` and because of it, your this selected link by default concatenated with `github` URL, hence I got `404` page error. 

```suggestion
[Live link](https://frolicking-brigadeiros-17734d.netlify.app)
```